src/main/java/data_access/InMemoryUserDAO.java

```diff
@@ -38,6 +39,9 @@ public void save(User user) {
     @Override
     public User get(String username) {
```

Comment:
Ak this point, I think we should rename this method to something more apt, like retrieveUser or userLookup, to distinguish it from the getCurrentUser method.

Reply:
Good idea, we can make a note of this for the final refactor.